.github/workflows/release.yml

```diff
@@ -275,19 +275,19 @@ jobs:
 
           # Remove backslashes before colons in commit details
```

Comment:
- The variable RELEASE_NOTES is being reassigned incorrectly. It should be updated to reflect the combination of existing content and new fix notes.